scripts/.gitignore

```diff
@@ -0,0 +1 @@
+.secrets
```

Comment:
This should be used and run on precommit. Raise a seperate issue for it.

Reply:
See #1082